addons/l10n_fi/data/account_tax_template.xml

```diff
@@ -0,0 +1,479 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="aland0" model="account.tax.template">
+            <field name="chart_template_id" ref="fi_chart_template"/>
+            <field name="name">Aland 0%</field>
+            <field name="description">Aland 0%</field>
+            <field name="amount">0.0</field>
+            <field name="type_tax_use">sale</field>
+            <field name="account_id" ref="chart2830"/>
+            <field name="refund_account_id" ref="chart2830"/>
+            <field name="amount_type">percent</field>
+            <field name="tag_ids" eval="[(6,0,[ref('304-sales0')])]"/>
+        </record>
+        <record id="purchase14euservice2" model="account.tax.template">
+            <field name="chart_template_id" ref="fi_chart_template"/>
+            <field name="name">Purchase 14% EU Service2</field>
```

Comment:
@thecodebasesite I think u need +14 and -14 tax.
If yes then use a group of tax.
like this

- Purchase 14% EU Service
     - (+) Purchase 14% EU Service
     - (-) Purchase 14% EU Service

Reply:
Do same in all (+)positive and (-)nagative tax

Reply:
@jva-odoo @mlaitinen Thank you for all of your improvement comments. We're currently working on these and we were wondering why the use of 'group of tax' is recommended? The naming change is obviously for higher descriptiveness, but is there some technical reason for the tax groups?

Reply:
@thecodebasesite I recommend a group of tax because if same positive and negative tax both always use in invoice or bill then it is complex to a user to select those two tax.
If you use a group of tax then it's the easiest way to do that.

If negative or positive same tax uses separately in invoice or bill then it's ok to not use a group of tax.

Reply:
@jva-odoo, one question regardin tax groups.
In our case negative taxes are never used directly by user, but they are used by fiscal postitions for tax reversals (i.e. EU purchases). In product we define regular tax like VAT 24%. This tax will be used for purchases and seales inside of the country. In EU sales (or purchases) to other countires than Finnland, we need to handle bills as tax free, but still report to tax authority substracted taxes. This is implemented in the fiscal position by mapping VAT 24% to two separate taxes negating each other (i.e. EU VAT +24% and EU VAT -24%). After that you have tax free vendor bill / invoice, but tax information is recorded for tax reporting to tax office.
In reporting we don't have ever to group positive and negative tax under same category, neither user has to manually handle positive and negative taxes. Do you still see it makes sense to add group to positive and negative taxes?

Reply:
@sprintit @thecodebasesite It is what is happening in other localizations for the intra-community too.  You make a tax (group) with two taxes of type none (which can not be used directly), one positive and one negative and you put purchase/sale on the tax that is the group.  In saas-12.3 we will translate it into one tax with repartition lines.

Reply:
Here's an example: https://github.com/odoo/odoo/pull/31016/files#diff-a40a9e6bbab80fe90d08dd1ba10e3ecbR53

This is how it's done and you also get the numbers correct on invoices and tax report when doing it like that. With the difference that in our example the `type_tax_use` is `purchase` and it'd be smarter to make it `none` like @jco-odoo mentioned.

I think the only downside for that is that then we need separate taxes for "Goods" and "Services" even for domestic taxes, which is quite pointless. It'd be awesome if fiscal position lines also had a "product type" field, so you could make some mapping rules only apply to "Service" type products for instance.

Reply:
@mlaitinen and @jco-odoo thank you for your input.
I see there is a possibility to set a group for a tax in field in tax_group_id
There is also a possibility to create a tax with amount_type "Group of taxes" and then add several taxes as children_tax_ids.

@jco-odoo, if I understood your comment correctly, you are suggesting creation of new tax with amount_tpye "Group of taxes" and not a tax group (account.tax.group).  Am I right?

@mlaitinen , I agree that your approach is appropriate and clear, but I didn't undestand your point about correct numbers on invoices and tax report. I did few tests and it looks like numbers in invoices look the same in both cases. I get same result when I group positive and negative taxes under "group of taxes" tax which I use in fiscal position or when I omit "group of taxes" tax and put both negative and positive taxes directly to fiscal position:
```
EU Fiscal position:

line1: Purchase 24% -> Purchase +24% Goods EU
line2: Purchase 24% -> Purchase -24% Goods EU
other lines...
```

After seeing how "group of taxes" works, I prefer your approuch though, but I wonder if I miss something. Should I get wrong numbers somewhere when I don't use "group of taxes"?

Reply:
Indeed, I was referring to the Group of Taxes.

Reply:
If you don't use the group of taxes, the user needs to put two taxes on the invoice line instead of one.

Reply:
This. And even though fiscal position might take care of that, the partners don't always have a fiscal position and it would be useful to be able to set the taxes manually (without having to remember to add the extra negative tax every time)